IntegrationWithPharmacies/UrgentMedicineOrderApi/Repository/UrgentMedicineOrderRepository.cs

```diff
@@ -1,28 +1,28 @@
-﻿using System.Collections.Generic;
-using System.Linq;
-using HealthClinic.CL.DbContextModel;
-using HealthClinic.CL.Model.Orders;
-
-namespace HealthClinic.CL.Repository
-{
-   public class UrgentMedicineOrderRepository : IUrgentMedicineOrderRepository
-    {
-        private MyDbContext dbContext;
-        public UrgentMedicineOrderRepository(MyDbContext dbContext)
-        {
-            this.dbContext = dbContext;
-        }
-        public UrgentMedicineOrderRepository() { }
-        public UrgentMedicineOrder Create(UrgentMedicineOrder order)
-        {
-            this.dbContext.UrgentMedicineOrder.Add(order);
-            this.dbContext.SaveChanges();
-            return order;
-        }
-
-        public List<UrgentMedicineOrder> GetAll()
-        {
-            return this.dbContext.UrgentMedicineOrder.ToList();
-        }
+﻿using System.Collections.Generic;
+using System.Linq;
+using UrgentMedicineOrderApi.DbContextModel;
+using UrgentMedicineOrderApi.Model;
+
+namespace UrgentMedicineOrderApi.Repository
+{
+   public class UrgentMedicineOrderRepository : IUrgentMedicineOrderRepository
+    {
+        private MyDbContext dbContext;

```

Comment:
Change name of MyDbContext property "dbContext" into "DbContext" so it starts with an upper letter.
Remove "this" in constructors and make it to be "DbContext = dbContext".
Remove  "this.dbContext" were possible and put "DbContext".